src/JZSolver.hpp

```diff
@@ -0,0 +1,16 @@
+#include "Algorithm.hpp"
+
+#define DEBUG 0
+
+class JZSolver : Algorithm
```

Comment:
Your algorithm solves only complete lines. Maybe rename it to FullLinesSolver.

Reply:
Why you named this class like this? I know that is your solution but I think the name of the class should describe the destiny...or is this short for something?

Reply:
This is just a first part of whole puzzle Solver. Project architecture allows us to create individual solvers for each of project's participants. I just gave mine individual name that consist of my name's initials and Solver suffix. I hope i will finish it on time ;)